### ISSUE TITLE
SmartCity-Accessibility Server Side Database Branch #31 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#306]
Build 'SmartCity-Accessibility Server Side Database Branch' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [Alexander Kaplan] 868acc867a3b499a32a72aabb105c88057f65045 - added abstracts and finished location manager abs #300

- [Alexander Kaplan] ca6421a081406fc4efc4d32bf0724183575eae55 - minor change to abstract location #300

- [Alexander Kaplan] 3fce3d5e01da9c3f3b0e2d6a908982df002a40a2 - remove use of Optional in user profile manager #294

- [Alexander Kaplan] 404c4ae31114484cf85aa897796c5422c96d9381 - added exception to database interface #295

- [Alexander Kaplan] a83fc3b5611c3cece3d3f0f971d277e6c4eace0f - added exception to database interface #295 for real

- [Alexander Kaplan] bf78c182d57e682d24f78f5d7f4fab87f4251419 - remove warnings

- [Alexander Kaplan] 99d256adf52b22bf503f1774bc16a951b3329759 - added abstract user profile manager #300

- [Alexander Kaplan] fba6c0e8aac11068715e3fd4989ef6a2db7b6a11 - added author credit to files

- [Alexander Kaplan] 53cb67eedb3416761a6c8a3868cdb2241b41b3b2 - abstract review manager #300

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Database%20Branch/31/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
